gpt4all-bindings/python/docs/gpt4all_desktop/cookbook/use-local-ai-models-to-privately-chat-with-Obsidian.md

```diff
@@ -0,0 +1,110 @@
+       </td>
+   </tr>
+   </table>
+
+2. **Add Collection**:
+   - Click on **+ Add Collection** to begin linking your Obsidian Vault.
+
+   <table>
+   <tr>
+       <td>
+           <!-- Screenshot of adding collection in LocalDocs -->
+           <img width="1348" alt="Screenshot of adding collection" src="https://raw.githubusercontent.com/nomic-ai/gpt4all/124ef867a9d9afd9e14d3858cd77bce858f79773/gpt4all-bindings/python/docs/assets/obsidian_adding_collection.png">
+
+
+   </td>
+   </tr>
+   </table>
+
+   - Name your collection
+
+
+3. **Create Collection**:
+   - Click **Create Collection** to initiate the embedding process. Progress will be displayed within the LocalDocs interface.
+
+4. **Access Files in Chats**:
+   - Load a model to chat with your files (Llama 3 Instruct is the fastest)
+   - In your chat, open 'LocalDocs' with the button in the top-right corner to provide context from your synced Obsidian notes.
+
+   <table>
+   <tr>
+       <td>
+           <!-- Screenshot of accessing LocalDocs in chats -->
+         <img width="1447" alt="Accessing LocalDocs in chats" src="https://raw.githubusercontent.com/nomic-ai/gpt4all/124ef867a9d9afd9e14d3858cd77bce858f79773/gpt4all-bindings/python/docs/assets/obsidian_docs.png">
+       </td>
+   </tr>
+   </table>
+
+5. **Interact With Your Notes:**
+   - Use the model to interact with your files
+   <table>
+   <tr>
+       <td>
+           <!-- Screenshot of interacting sources -->
+         <img width="662" alt="osbsidian user interaction" src="https://raw.githubusercontent.com/nomic-ai/gpt4all/124ef867a9d9afd9e14d3858cd77bce858f79773/gpt4all-bindings/python/docs/assets/osbsidian_user_interaction.png">
+       </td>
+   </tr>
+   </table>
+   <table>
+   <tr>
+       <td>
+           <!-- Screenshot of viewing sources -->
+         <img width="662" alt="osbsidian GPT4ALL response" src="https://raw.githubusercontent.com/nomic-ai/gpt4all/124ef867a9d9afd9e14d3858cd77bce858f79773/gpt4all-bindings/python/docs/assets/obsidian_response.png">
+       </td>
+   </tr>
+   </table>
+
+7. **View Referenced Files**:
+   - Click on **Sources** below LLM responses to see which Obsidian Notes were referenced.
+
+   <table>
+   <tr>
+       <td>
+           <!-- Referenced Files  -->
+         <img width="643" alt="Referenced Files" src="https://raw.githubusercontent.com/nomic-ai/gpt4all/124ef867a9d9afd9e14d3858cd77bce858f79773/gpt4all-bindings/python/docs/assets/obsidian_sources.png">
+       </td>
+   </tr>
+   </table>
+
+## How It Works
+
+Obsidian for Desktop syncs your Obsidian notes to your computer, while LocalDocs integrates these files into your LLM chats using embedding models. These models find semantically similar snippets from your files to enhance the context of your interactions.
+
+To learn more about embedding models and explore further, refer to the [Nomic Python SDK documentation](https://docs.nomic.ai/atlas/guides/embeddings).
```

Comment:
this link is broken, let's change it to https://docs.nomic.ai/atlas/capabilities/embeddings

Also @AndriyMulyar I will look into proper link redirecting long term, both to deal with potential docs links changes and to improve SEO